tests/network/vmi_networking.go

```diff
@@ -267,22 +267,22 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			bridgeIface.PciAddress = "0000:03:00.0"
 
 			bridgeNetwork := libvmi.MultusNetwork(secondaryNetName, nadName)
-			e1000VMI := libvmifact.NewAlpine(
+			multiIfaceVMI := libvmifact.NewAlpine(
```

Comment:
How about just `vmi`?
Additionally, consider squashing this commit with the previous one.

Reply:
> > How about just `vmi`?
> 
> Why ?

Because IMO `multiIfaceVMI` does not add additional info in the context of this test.
But I won't insist.

Reply:
I also prefer `vmi` and it is consistent with other tests which do not express in the name what it contains, but what is their purpose (e.g. server, client, etc...). In this case, we have only one, I see no reason to break convention.